### PR TITLE
User Stories

### PR DESCRIPTION
**Pull request by nadavc-jfrog**

### CODE REVIEW

File: user-stories.html [hunk 1]
```diff
@@ -0,0 +1 @@
...l="noopener noreferrer" taget="_blank">Slack #conan</a></li></ul></div></div></div></div></div></div></footer><div id="openSourceSubscribeModal" class="modal px-2 fade" aria-hidden="true" aria-labelledby="modalSuccessTitleFH" role="dialog" tabindex="-1"><div class="modal-dialog" role="document"><div class="modal-content"><div class="modal-body p-0 position-relative"><button type="button" class="close position-absolute" data-dismiss="modal" aria-label="Close">X</button><div class="splitter d-flex h-100 flex-wrap"><div class="left d-flex justify-content-center align-items-center text-center p-3"><div><p class="top text-white"><strong>Thank You For Downloading</strong></p><p class="top text-white">Your download should<br>begin immediately.</p></div></div><div class="right d-flex justify-content-center text-center text-lg-left align-items-center p-4"><div class="mx-auto"><h2 id="modalSuccessTitleFH">Stay Up to Date</h2><p class="open-sans">Sign up for product updates and new features.</p><div class="os-form"><form id="mktoForm_3511"></form><p id="subscribeSuccess" class="black bold mb-0"></p></div></div></div></div></div></div></div></div><script src="js/all.js"></script></body></html>
```
Comment (tapia) [inline user-stories.html:1]: This file is generated, shouldn't be added to the repo

File: user-stories/rti.html [hunk 2]
```diff
@@ -0,0 +1 @@
...noopener noreferrer" taget="_blank">Slack #conan</a></li></ul></div></div></div></div></div></div></footer><div id="openSourceSubscribeModal" class="modal px-2 fade" aria-hidden="true" aria-labelledby="modalSuccessTitleFH" role="dialog" tabindex="-1"><div class="modal-dialog" role="document"><div class="modal-content"><div class="modal-body p-0 position-relative"><button type="button" class="close position-absolute" data-dismiss="modal" aria-label="Close">X</button><div class="splitter d-flex h-100 flex-wrap"><div class="left d-flex justify-content-center align-items-center text-center p-3"><div><p class="top text-white"><strong>Thank You For Downloading</strong></p><p class="top text-white">Your download should<br>begin immediately.</p></div></div><div class="right d-flex justify-content-center text-center text-lg-left align-items-center p-4"><div class="mx-auto"><h2 id="modalSuccessTitleFH">Stay Up to Date</h2><p class="open-sans">Sign up for product updates and new features.</p><div class="os-form"><form id="mktoForm_3511"></form><p id="subscribeSuccess" class="black bold mb-0"></p></div></div></div></div></div></div></div></div><script src="../js/all.js"></script></body></html>
```
Comment (tapia) [inline user-stories/rti.html:1]: This directory (`user-stories`) is generated, and shouldn't be added to the repo

**Review by tapia — CHANGES_REQUESTED** [review]
Please be careful not to add generated content to the repo

**Comment by nadavc-jfrog** [comment]
@tapia - they are already in `.gitignore`, you have to remove these files from the repo if you don't want them to be tracked.
Please see - https://stackoverflow.com/questions/45400361/why-is-gitignore-not-ignoring-my-files

**Comment by tapia** [comment]
@nadavc-jfrog Yes, the files mentioned should be removed from the repo, but the reviewer isn't supposed to make changes to the PR, it should be the author (@ncamaa if I'm not mistaken) who fixes it.

File: user-stories.html (hunk 1 above)
Comment (nadavc-jfrog) [reply user-stories.html]: @tapia removed.

File: user-stories/rti.html (hunk 2 above)
Comment (nadavc-jfrog) [reply user-stories/rti.html]: @tapia removed.

**Pull request merged**